docs/csharp/programming-guide/classes-and-structs/how-to-initialize-a-dictionary-with-a-collection-initializer.md

```diff
@@ -12,6 +12,16 @@ ms.assetid: 25283922-f8ee-40dc-a639-fac30804ec71
 
 A <xref:System.Collections.Generic.Dictionary%602> contains a collection of key/value pairs. Its <xref:System.Collections.Generic.Dictionary%602.Add%2A> method takes two parameters, one for the key and one for the value. One way to initialize a <xref:System.Collections.Generic.Dictionary%602>, or any collection whose `Add` method takes multiple parameters, is to enclose each set of parameters in braces as shown in the following example. Another option is to use an index initializer, also shown in the following example.
 
+> NOTE: The major difference between these two ways of initializing the collection is that in case of having multiplied keys, for example:
```

Comment:
nit: word choice:

```suggestion
> NOTE: The major difference between these two ways of initializing the collection is that in case of having duplicated keys, for example:
```